src/clusterfuzz/_internal/cron/external_testcase_reader.py

```diff
@@ -0,0 +1,131 @@
+# Copyright 20204 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Automated ingestion of testcases via IssueTracker."""
+
+import re
+import requests
+
+from appengine.libs.gcs import prepare_blob_upload
+from appengine.libs.form import generate_csrf_token
+from clusterfuzz._internal.issue_management.google_issue_tracker.issue_tracker \
```

Comment:
Style guide says to import modules not classes or functions.

Reply:
ooh noted!

Reply:
I think you need to do this instead.
`from clusterfuzz._internal.issue_management.google_issue_tracker import issue_tracker`

Reply:
bless